libs/jump_portrait/src/jump_portrait/fetch.py

```diff
@@ -373,3 +362,11 @@ def get_gene_images(
     )
 
     return images
+
+metadata_pre = get_item_location_info("MYT1")
```

Comment:
I think these were not supposed to be here. These are the lines for testing from the readme, right?